src/index.ts

```diff
@@ -16,6 +17,7 @@ app.use(express.json());
 app.use("/login", loginRouter);
 app.use("/register", registerRouter);
 app.use("/api", authenticateUser);
+app.use("/api/validate-admin", authenticateUser, validateAdmin);
```

Comment:
line 19 should look like this:
app.use("/api", authenticateUser, authenticateUser, validateAdmin, adminEndpoint);
delete line 20 and create adminEndpoint